lib/jekyll-plugin-platoniq-journal/quote_tag.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Jekyll
+  class QuoteBlockTag < Liquid::Block
+    def initialize(tag_name, input, tokens)
+      super
+      @input = input
+    end
+
+    def render(_context)
+      text = super
+
+      jdata = if !@input.nil? && !@input.empty?
+                JSON.parse(@input)
+              else
+                JSON.parse({ :example => "123" })
+              end
+
+      output = []
+
+      output << %(<figure class="pj-quote">)
+      output << <<~QUOTE
+        <blockquote>
+          #{text}
+        </blockquote>
+        <figcaption class="pj-quote--author">
+          by #{jdata["author"]}
```

Comment:
@agustibr I still don't understand how to translate the string "by" in this custom tag, and the same goes for including an svg stored as `_includes/svg/icon-quotes.liquid` in the https://github.com/Platoniq/jekyll-theme-platoniq-journal repo, sorry, I couldn't find any examples (I only have the CTA and Numbers tags to rely on and none of those have either translated strings or SVGn includes)

Reply:
No worries, I'll try to answer this tomorrow 😄